.github/workflows/swift-toolchain.yml

```diff
@@ -939,55 +948,85 @@ jobs:
         uses: hendrikmuhs/ccache-action@2e0e89e8d74340a03f75d58d02aae4c5ee1b15c6
         with:
           max-size: 500M
-          key: ${{ steps.workspace_hash.outputs.hash }}-windows-${{ matrix.arch }}-compilers
+          key: ${{ steps.workspace_hash.outputs.hash }}-${{ matrix.os }}-${{ matrix.arch }}-compilers
           variant: sccache
 
       - name: Configure Compilers
         env:
           NDKPATH: ${{ steps.setup-ndk.outputs.ndk-path }}
         run: |
-          if ( "${{ matrix.arch }}" -eq "arm64" ) {
-            $CMAKE_SYSTEM_NAME="-D CMAKE_SYSTEM_NAME=Windows"
-            $CMAKE_SYSTEM_PROCESSOR="-D CMAKE_SYSTEM_PROCESSOR=ARM64"
-            $CACHE="Windows-aarch64.cmake"
-
-            # FIXME(compnerd) re-enable runtimes after we sort out compiler-rt
-            (Get-Content ${{ github.workspace }}/SourceCache/swift/cmake/caches/Windows-aarch64.cmake).Replace(' runtimes', '') | Set-Content ${{ github.workspace }}/SourceCache/swift/cmake/caches/Windows-aarch64.cmake
-          } else {
-            $CACHE="Windows-x86_64.cmake"
+          if ( "${{ matrix.os }}" -eq "Windows" ) {
+            $SWIFTC = cygpath -m (Get-Command swiftc).Source
+            # Use toolchain clang to avoid broken __prefetch intrinsic on arm64 in Clang 18.
```

Comment:
Can we tie an issue to this? We should switch back as this should be fixed in clang 19 I suspect.

Reply:
Filed https://github.com/compnerd/swift-build/issues/846 and added a comment.